tools/clang/test/SemaHLSL/ds-error-outputpatch-size.hlsl

```diff
@@ -1,5 +1,4 @@
 // RUN: %dxc -T ds_6_0 -E main -verify %s
-// RUN: %dxc -T ds_6_0 -E main -verify %s -spirv
```

Comment:
I think merging or rebasing will make it so this diff doesn't show in your PR.

Reply:
That was an accident, will be fixed on next commit.